addons/website_sale/views/templates.xml

```diff
@@ -567,20 +567,21 @@
                     <div id="o_wsale_offcanvas_pricelist"
                         class="accordion-collapse collapse"
                         aria-labelledby="o_wsale_offcanvas_orderby_header">
-                        <div class="accordion-body pt-0">
+                        <div class="accordion-body">
                             <div class="list-group list-group-flush">
                                 <a t-foreach="website_sale_pricelists" t-as="pl"
                                     role="menuitem"
                                     rel="noindex,nofollow"
                                     t-att-href="'/shop/change_pricelist/%s' % pl.id"
-                                    class="list-group-item border-0 ps-0 pb-0"
+                                    class="list-group-item mb-1 p-0 border-0"
```

Comment:
Should we add `mx-n1 px-1` as for [`l846`](https://github.com/odoo/odoo/pull/144186/files#diff-635040c730ccbb8e4f6f2bf6472cf48a0cc9b8a3d488289dabdfce8096811915R846)? (same for other occurences of this line under)